pkg/ali/components/Networking.vue

```diff
@@ -0,0 +1,601 @@
+        });
+      } catch (err) {
+        const parsedError = err.error || '';
+
+        this.$emit('error', this.t('ack.errors.vpcs', { e: parsedError || err }));
+      }
+      this.loadingVPCs = false;
+    },
+    async getVswitches() {
+      this.loadingVswitches = true;
+      this.allVSwitches = {};
+      const { resourceGroupId, vpcId } = this.config;
+
+      try {
+        const res = await getAlibabaVSwitches(this.$store, this.alibabaCredentialSecret, this.regionId, vpcId, resourceGroupId );
+        const vSwitches = res?.VSwitches?.VSwitch || [];
+
+        vSwitches.forEach((vswitch) => {
+          let label = vswitch.VSwitchName ? `${ vswitch.VSwitchName } (${ vswitch.VSwitchId })` : vswitch.VSwitchId;
+
+          if (vswitch.ZoneId) {
+            label += ` - ${ vswitch.ZoneId }`;
+          }
+          if (vswitch.CidrBlock) {
+            label += ` - ${ vswitch.CidrBlock }`;
+          }
+          this.allVSwitches[vswitch.VSwitchId] = {
+            label, cidr: vswitch.CidrBlock, zoneId: vswitch.ZoneId
+          };
+        });
+      } catch (err) {
+        const parsedError = err.error || '';
+
+        this.$emit('error', this.t('ack.errors.vswitches', { e: parsedError || err }));
+      }
+      this.loadingVswitches = false;
+    },
+    async getZones() {
+      this.loadingAvailabilityZones = true;
+      this.allAvailabilityZones = [];
+      try {
+        const res = await getAlibabaZones(this.$store, this.alibabaCredentialSecret, this.regionId );
+        const zones = (res?.Zones?.Zone || []).map((zone) => ({ value: zone.ZoneId, label: zone.LocalName }));
+
+        this.allAvailabilityZones = zones;
+        if (this.isNewOrUnprovisioned && !this.chooseVPC) {
+          this.config.zoneIds = zones.map((z) => z.value);
+        }
+      } catch (err) {
+        const parsedError = err.error || '';
+
+        this.$emit('error', this.t('ack.errors.zones', { e: parsedError || err }));
+      }
+      this.loadingAvailabilityZones = false;
+    },
+    zonesChanged() {
+      let zones = [];
+
+      if (!this.chooseVPC) {
+        zones = this.config.zoneIds;
+      } else if (this.config.vswitchIds) {
+        this.config.vswitchIds.forEach((vswitchId) => {
+          const vswitch = this.allVSwitches[vswitchId];
+
+          zones.push(vswitch.zoneId);
+        });
+      }
+      this.$emit('zoneChanged', new Set(zones));
+    }
+
+  }
+});
+</script>
+
+<template>
+  <div
+    class="mt-10"
+    data-testid="ack-networking"
+  >
+    <div
+      class="row mb-10"
+    >
+      <div class="col span-4">
+        <LabeledSelect
+          v-model:value="config.resourceGroupId"
+          data-testid="ack-networking-resourceGroupId"
+          :mode="mode"
+          :options="resourceGroupOptions"
+          label-key="ack.networking.resourceGroupId.label"
+          option-key="value"
+          option-label="label"
+          :loading="loadingResourceGroups"
+          required
```

Comment:
This is marked as required but "Not Selected" is also an option. Is "Not Selected" a valid choice? If yes should probably use the generic translation "None" and this input shouldn't be marked required